File: PartnerApi/Swift/b2_list_group_members.swift
import Foundation

let apiUrl = ""  // Provided by b2_authorize_account
let accountAuthorizationToken = ""  // Provided by b2_authorize_account
let adminAccountId = ""  // Provided by b2_authorize_account
let groupId = ""  // Provided by b2_list_groups

let postData: [String: Any] = [
  "adminAccountId": adminAccountId,
  "groupId": groupId,
]

var request = URLRequest(url: URL(string: "\(apiUrl)/b2api/v4/b2_list_group_members")!)
request.httpMethod = "POST"
request.setValue(accountAuthorizationToken, forHTTPHeaderField: "Authorization")
request.setValue("application/json", forHTTPHeaderField: "Content-Type")
request.httpBody = try? JSONSerialization.data(withJSONObject: postData)

let (data, response) = try await URLSession.shared.data(for: request)

print(String(decoding: data, as: UTF8.self))

File: PartnerApi/Python/b2_list_group_members.py
import json
import requests

api_url = ""  # Provided by b2_authorize_account
account_authorization_token = ""  # Provided by b2_authorize_account
admin_account_id = ""  # Provided by b2_authorize_account
group_id = ""  # Provided by b2_list_groups

response = requests.post(
    f"{api_url}/b2api/v4/b2_list_group_members",
    headers={"Authorization": account_authorization_token},
    json={"adminAccountId": admin_account_id, "groupId": group_id},
)
response_data = response.json()
print(json.dumps(response_data, indent=2))
